scss/_base_typography.scss

```diff
@@ -174,177 +99,51 @@
   h6 {
     font-family: unquote($font-heading-family);
     font-weight: 300;
-    line-height: 1.3;
+    line-height: 1.5;
+    margin-bottom: .5rem;
   }
 
-  h1,
-  .heading-1 {
-    font-size: 2.8125em;
-    margin-bottom: .5em;
+  h1 {
+    font-size: 180%;
```

Comment:
I left these here to standardise cross-browser inconsistencies but maybe that's not much of a concern - I'll remove.